docs/180-microsoft-integration/010-powerapps.md

```diff
@@ -5,5 +5,29 @@ parent: Microsoft Integration
 permalink: /microsoft-integration/powerapps
 nav_order: 40
 has_children: true
-tags: ["integration", "microsoft", "powerapps", "dataverse"]
----
\ No newline at end of file
+tags: ["integration", "microsoft", "powerapps", "dataverse", "powerautomate"]
+---
+
+## Introduction
+We now offering you an in-app experience for Power Apps and Power Automate where you can build your workflows directly within CluedIn or manage your Master Data directly in Power Apps.
+
+## Capabilities
```

Comment:
Please add a description of this section. For example: This section describes the benefits of integration with PowerApps and Power Automate.